### ISSUE TITLE
Less helpful error message when source address of --sourceaddr cannot be used

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#38]
Zonemaster CLI has the option --sourceaddr. If that is set to localhost or to an adress not bound to any interface, the error message is less helpful.

```
$ ifconfig eth0 | grep "inet "
          inet addr:109.74.12.66  Bcast:109.74.13.255  Mask:255.255.254.0

$ zonemaster-cli --sourceaddr 109.74.12.67 google.de
Seconds Level     Message
======= ========= =======
   0.15 ERROR     No response from nameserver for zone . when trying to fetch glue.
   0.17 CRITICAL  Not enough data about google.de was found to be able to run tests.

$ zonemaster-cli --sourceaddr 127.0.0.1 google.de
Seconds Level     Message
======= ========= =======
   0.14 ERROR     No response from nameserver for zone . when trying to fetch glue.
   0.15 CRITICAL  Not enough data about google.de was found to be able to run tests.

$ zonemaster-cli --sourceaddr ::1 google.de
Seconds Level     Message
======= ========= =======
  90.32 ERROR     No response from nameserver for zone . when trying to fetch glue.
  90.34 CRITICAL  Not enough data about google.de was found to be able to run tests.
```

**NOTE** (2021-01-19): #110 fixes the first case (using a address not attached to any interface) but not the second and third (using a localhost address). The first case was the most important, so this issue is considered to be resolved.

**Comment by sandoche2k** [comment]
@matsduf can you propose a text how the error message should be ?

**Comment by matsduf** [comment]
"Address ADDR cannot be used as source address for DNS queries."

`zonemaster-cli` should stop at such error.

**Comment by mattias-p** [comment]
This is an input validation issue for the command line interface.

**Comment by matsduf** [comment]
PR created in #110.

**Comment by matsduf** [comment]
Resolved by #110.
